assembl/static2/.storybook/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
Why is this file needed in folder .storybook?

Reply:
Oh is it because storybook runs its own web server?

Reply:
yes, it starts its own webpack server on port 9001